content/md/en/docs/tutorials/connect-other-chains/local-relay.md

```diff
@@ -0,0 +1,178 @@
+
+- [_Plain_ `rococo-local` **relay** chain spec](https://github.com/substrate-developer-hub/substrate-docs/blob/main-md/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-plain.json)
+- [_Raw_ `rococo-local` **relay** chain spec](https://github.com/substrate-developer-hub/substrate-docs/blob/main-md/static/assets/tutorials/cumulus/chain-specs/rococo-custom-2-raw.json)
+
+Plain chain spec files are in a more human readable and modifiable format for your inspection.
+You will need to convert it to a SCALE encoded **raw** chain spec to use when starting your nodes.
+Jump to the [raw chainspec generation](/tutorials/connect-other-chains/connect-parachain/#configure-a-parachain-for-a-specific-relay-chain-and-para-id) section to see how to do that.
+
+The above raw chain specs were created according to the steps in the [create your own chain spec](/tutorials/get-started/trusted-network/#create-a-custom-chain-specification) section.
+If you need more validators to add additional parachains to your relay chain, or want to use custom non-development keys for example, you _must_ create a custom chain specification to fit your needs.
+
+## Start your relay chain
+
+Before we can start block production for a parachains, we need to launch a relay chain for them to connect to.
+This section describes in detail how to start both nodes using the above [two-validator raw chain spec](/assets/tutorials/cumulus/chain-spec/rococo-custom-2-raw.json) as well as the general instructions for starting additional nodes.
+
+### Start the `alice` validator
+
+```bash
+# Start Relay `Alice` node
+./target/release/polkadot \
+--alice \
+--validator \
+--base-path /tmp/relay/alice \
+--chain <path to spec json> \
+--port 30333 \
+--ws-port 9944
+```
+
+The port (`port`) and websocket port (`ws-port`) specified here are the defaults and thus those flags can be omitted.
+However we choose to leave them in to enforce the habit of checking their values.
+Once this node is launched, no other nodes on the same local machine can use these ports.
+
+When the node starts you will see several log messages. **Take note of the node's Peer ID** in the logs.
+We will need it when connecting other nodes to it.
+It will look something like this:
+
+```bash
+🏷 Local node identity is: 12D3KooWGjsmVmZCM1jPtVNp6hRbbkGBK3LADYNniJAKJ19NUYiq
+```
+
+### Start the `bob` validator
+
+```bash
+./target/release/polkadot \
+--bob \
+--validator \
+--base-path /tmp/relay-bob \
+--chain <path to spec json> \
+--bootnodes /ip4/<Alice IP>/tcp/30333/p2p/<Alice Peer ID> \
+--port 30334 \
+--ws-port 9945
+```
+
+Bob's command is perfectly analogous to Alice's.
+It differs from Alice's one by his own base path, his own validator key (`--bob`), and his own ports.
+Finally he adds a `--bootnodes` flag.
+This flag is not strictly necessary if you are running the entire network on a single local machine, but it is necessary when operating over the network.
+
+<!-- TODO NAV.YAML -->
+<!-- add these back -->
```

Comment:
```suggestion
```

Reply:
This page is not included yet in the Nav and thus not ready to be uncommented. Once readythis section can  go live. Not till then.